src/main/java/com/epam/lowcost/controller/UserController.java

```diff
@@ -130,7 +149,7 @@ public String deleteUser(@ModelAttribute(value = "sessionUser") User sessionUser
             return "redirect:" + TICKETS + SELF;
         }
         model.addAttribute("message", userService.deleteUser(id));
-        return "users";
+        return "redirect:/user/all/1";
```

Comment:
?

Reply:
I will change it to constants if everything else is ok. For now I used string for simplicity.

Reply:
no, it is a bad practice. Doing whole task then make pull request

Reply:
Changed redirects to use constants.